as2/server/redemption.md

```diff
@@ -0,0 +1,15 @@
+# Redemption
+##### Messages pertaining to the redemption server.
+---
+### Join Server
+Sent to the penguin trying to log in to the redemption server to give needed player information. `rjs`
+
+|Argument|Type|Variadic?|Default
+|---|---|---|---
+|Redemeed Books|String|Yes|NULL
+|Unknown & Not Needed|Unsigned integer|No|0
+|Member?|Unsigned integer|No|0
```

Comment:
You can just change this to maybe "Boolean integer (0, 1)" and most people will understand without the need for the note. But it is good to be explicit.